src/Symfony/Component/Yaml/Parser.php

```diff
@@ -359,6 +299,31 @@ public function parse($value, $flags = 0)
             mb_internal_encoding($mbEncoding);
         }
 
+        if (array_key_exists('<<', $data)) {
```

Comment:
But above we always ensure that we add an array for the `<<` key. So if we want to throw an exception for `null`, we must do it there.